massa-network/src/node_worker.rs

```diff
@@ -26,14 +30,21 @@ use tokio::{
 };
 use tracing::{debug, trace, warn};
 
+#[derive(Debug, Serialize, Deserialize)]
+pub enum ToSend {
+    Msg(Message),
+    Block(BlockId),
```

Comment:
Maybe call this variant BlockId ?

Reply:
I think `Block` is more descriptive, because the point of the message is to actually send the block over the network.

Reply:
Ok